services/tokei/tokei-github-lines.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
```

Comment:
Few other thoughts below that could impact the desired filename, but either way I think we'll definitely want to drop `github` as there's nothing github specific about the class/badge here

Reply:
I think `tokei.service.js` will probably be the best bet

Reply:
Oh yeah, forgot to change that name! It was originally for just GitHub projects but then I decided to expand it a bit. I think `tokei-lines.service.js` might be the better option, though, because tokei can do more than just count lines of code (i.e. count of comments and individual files are also included in the API response). 

Maybe in the future, someone could implement a badge for a specific file as well? (in that case doesn't it make more sense to use `tokei-lines.service.js`?)

Reply:
Renamed.